docs/integrations/app-integrations/slack.mdx

```diff
@@ -155,111 +153,116 @@ The following usage applies when **Connection Method 2** was used to connect Sla
 See the usage for **Connection Method 1** [via the `CREATE CHATBOT` syntax](/sql/tutorials/create-chatbot).
 </Warning>
 
-You can select all messages from a conversation using the below query.
+Retrieve data from a specified table by providing the integration and table names:
 
 ```sql
 SELECT *
-FROM mindsdb_slack.messages
-WHERE channel_id="<channel-id>";
+FROM slack_datasource.table_name
+LIMIT 10;
 ```
 
-<Tip>
-To find the channel ID of a conversation, you can use the `conversations` table:
-
-```sql
-SELECT *
-FROM mindsdb_slack.conversations
-WHERE name = "<channel-name>";
-```
+## Supported Tables
 
-Please note that if your workspace has more than 1000 conversations, you may need to use the `LIMIT` clause to retrieve all conversations. More information on this can be found below.
+The Slack integration supports the following tables:
 
-You can also find the channel ID by right-clicking on the conversation in Slack, selecting 'View conversation details' or 'View channel details,' and copying the channel ID from the bottom of the 'About' tab.
-</Tip>
+### `conversations`
```

Comment:
Maybe we can add `Table` to the title too as `conversations` table

Reply:
Done.